tests/go.rs

```diff
@@ -0,0 +1,134 @@
+use ansi_term::Color;
```

Comment:
This is more a note to myself. The way that we integration test, we don't test for a case where the module _shouldn't_ appear. As it exists, `common:render_module()` wouldn't let us check that, but it'd be worth making it work that way.

@johnletey is this something you want to tackle, or should I address this another time?

Reply:
@matchai If you don't mind, I'd prefer if you addressed this another time